mlir-clang/Lib/clang-mlir.cc

```diff
@@ -985,106 +985,68 @@ ValueWithOffsets MLIRScanner::VisitDoStmt(clang::DoStmt *fors) {
 ValueWithOffsets MLIRScanner::VisitOMPParallelForDirective(
     clang::OMPParallelForDirective *fors) {
   IfScope scope(*this);
-  fors->dump();
```

Comment:
As a general remark, what do you think to split the OMP support on its own file? Maybe a simple class that derives the MLIRScanner? If you like the idea I can try something on Monday.

I also merged the renaming of ValueWithOffsets, so we should update this patch too.